### DISCUSSION TITLE
How can I add prisma to dependencies instead of devDependencies?

### DISCUSSION THREAD

**Discussion by wu-jiawei** [prisma/prisma#19668] [Q&A]
We're using Google Cloud Build to deploy a nestjs app to GKE. We build the image in the build step and then deploy the image to GKE in a separate deploy step, after that the migrations are intended to be run. But when running migrations using prisma we got
errors saying that prisma not found. The reason is that in the build step prisma was installed automatically to devDependencies even if it was added to dependencies in package.json, and then pnpm automatically uninstalls packages in devDependencies. 
Therefore, we are wondering if there is any way to install prisma to dependencies?
We are using prisma@4.15.0 and pnpm@7.5.0.

**Comment by janpio** [comment]
> The reason is that in the build step prisma was installed automatically to devDependencies even if it was added to dependencies in package.json, and then pnpm automatically uninstalls packages in devDependencies.

That should not be happening. If it is, please open an issue so we can investigate.
